tests/ops/test_channel_ops.py

```diff
@@ -120,6 +123,48 @@ def test_gamma_arbitrary(self, tol):
         assert np.allclose(op(0.1, wires=0).kraus_matrices, expected, atol=tol, rtol=0)
 
 
+class TestBitFlipChannel:
+    """Tests for the quantum channel BitFlipChannel"""
+
+    def test_p_zero(self, tol):
+        """Test p=0 gives correct Kraus matrices"""
+        op = channel.BitFlipChannel
+        assert np.allclose(op(0, wires=0).kraus_matrices[0], np.eye(2), atol=tol, rtol=0)
+        assert np.allclose(op(0, wires=0).kraus_matrices[1], np.zeros((2, 2)), atol=tol, rtol=0)
+
+    def test_p_arbitrary(self, tol):
```

Comment:
This is fine, but it's good practice to run tests for several parameter values. This is usually done using the parametrize decorator
```python
@pytest.mark.parametrize("p", [0.1, 0.4, 0.7])
def test_p_arbitrary(self, p, tol)
```

Reply:
Good call, this allows for merging the two separate tests into a single parameterized test.